### DISCUSSION TITLE
is there a plan about when to publish v7.0.0?

### DISCUSSION THREAD

**Discussion by shawumu** [Turfjs/turf#2491] [Q&A] [answered]
I notice that v7.0.0 planed fetures has completed in 82%，and v7.0.0-alpha.0 is publihsed last year. I want to know how far away from v7.0.0 be published?  how much work needs be done？ If I want to help, what can I do ?

**Comment by twelch** [comment, marked as answer]
Hi @shawumu, thanks for asking. There is not a plan for when exactly to publish v7, progress is slow.  For reference here's the roadmap - https://github.com/Turfjs/turf/milestone/19.

7.0.0-alpha.1 was published just yesterday - [7.0.0-alpha.1](https://www.npmjs.com/package/@turf/turf/v/7.0.0-alpha.1).  The changelog is up to date.

The easiest way to help if you're serious is to find an issue you can pick up and communicate your interest and intent.  You could:
- confirm that merged 7.0 fixes are working as expected in the most recent alpha release and communicate that back in the relevant ticket.  This would also help others following the ticket become aware that there is an alpha release available they can try out.
- if you're a self-starter and comfortable with the tooling or learning it, you can clone and check out the turf codebase on your own and learn how the monorepo is setup, and how to build and run tests.  Then find something to work on.
- if you don't know what to do on a given ticket, you could research the issue at least and share findings to help find a way forward.
- learn how the doc system works and contribute to doc related tickets.
- you could also pick up newer tickets that are not in 7.0 but would benefit from being fixed.

**Comment by smallsaucepan** [comment]
Hi @shawumu. For your information, Turf v7 has now been officially released.
